melosys-eessi-app/src/main/java/no/nav/melosys/eessi/config/MDCLogging.java

```diff
@@ -15,4 +20,20 @@ public static void loggSedID(String sedID) {
     public static void slettSedIDLogging() {
         MDC.remove(SED_ID);
     }
+
+    public static void loggCorrelationId() {
+        MDC.put(CORRELATION_ID, UUID.randomUUID().toString());
+    }
```

Comment:
Navngivingen er litt uheldig på denne. Denne logger ingenting, men lager en ny correlationID som erstatter tidligere, hvis det har vært noen tidligere. Hvorfor er dette nødvendig? Fikser ikke interceptor/filter dette for oss? 
Hvis det er slik at dette bare er brukt i de tilfellene prosess kjører, så burde vi kanskje navngi det litt bedre så når man leser koden forstår man at vi setter opp en correlation ID 🤔

Reply:
Dette er MDCOperations i Api. Så endrer navn til det samme. Noen ganger må det settes manuelt, i eessi ifm mottak av kafka meldinger. Jeg fant ikke noe tilsvarende som interceptorer/filter for kafka listener